sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosContainerChangeFeedTest.java

```diff
@@ -70,7 +70,7 @@
     private final Multimap<String, ObjectNode> partitionKeyToDocuments = ArrayListMultimap.create();
     private final String preExistingDatabaseId = CosmosDatabaseForTest.generateId();
 
-    @Factory(dataProvider = "simpleClientBuildersWithDirect")
+    @Factory(dataProvider = "simpleClientBuildersWithDirectTcp")
```

Comment:
You can just change the implementation of `simpleClientBuildersWithDirect` to remove DirectHttps that will be easier to do as you don't have to touch every test file to change `simpleClientBuildersWithDirect` to `simpleClientBuildersWithDirectTcp`.

the change will become mostly isolated only in one or two place

Reply:
Makes sense, will do that.

Reply:
Updated.